packages/patrol_gen/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: patrol_gen
+description: >
+  Generate contracts for all languages
+version: 1.0.0
+homepage: https://patrol.leancode.co
+repository: https://github.com/leancodepl/patrol
+issue_tracker: https://github.com/leancodepl/patrol/issues
+
+environment:
+  sdk: '>=2.18.0 <4.0.0'
+
+dependencies:
+  meta: ^1.7.0
+  path: ^1.8.2
+  analyzer: ^6.1.0
+  dart_style: ^2.3.2
+  
+dev_dependencies:
+  leancode_lint: ^3.0.0
```

Comment:
leancode_lint is added to pubspec but there's no analysis_options.yaml in the `patrol_gen` package. Is this intentional? Does the analysis work? (I suspect not because I noticed a few small things (e.g. unnecessary var) that leancode_lint would catch)

Reply:
I forgot to copy&paste